### PR TITLE
Ruby SDK v0.0.1

### PR DESCRIPTION
**Pull request by eahefnawy**

### CODE REVIEW

File: lib/yellow-sdk/client.rb [hunk 1]
```diff
@@ -11,49 +11,56 @@ def initialize(api_key, api_secret)
             @api_secret = api_secret
         end
 
-        def create_invoice(base_ccy, base_price, callback)
+        def create_invoice(hash={})
```
Comment (eahefnawy) [inline lib/yellow-sdk/client.rb:14]: this is the standard approach of dynamic arguments in ruby. Starting from Ruby v2.0.0 we can use `**kwargs` just like in python, but it seems that approach is still not that common yet. And many ruby devs still use Ruby 1.9. So I think this is good to go.

File: lib/yellow-sdk/client.rb [hunk 2]
```diff
@@ -74,6 +80,8 @@ def make_request(method, endpoint, body)
             request.add_field('API-Key', @api_key)
             request.add_field('API-Nonce', nonce)
             request.add_field('API-Sign', signature)
+            request.add_field('API-Platform', "#{RUBY_PLATFORM} - Ruby #{RUBY_VERSION}")
```
Comment (eahefnawy) [inline lib/yellow-sdk/client.rb:83]: RUBY_PLATFORM constant should return the OS release when applicable. So it's a little different than python.

File: lib/yellow-sdk/client.rb (hunk 1 above)
Comment (JamesPiechota) [inline lib/yellow-sdk/client.rb:14]: Sounds good

**Comment by JamesPiechota** [comment]
LGTM

**Comment by JamesPiechota** [comment]
Let me know when the demo is done and I can update the docs and we can let merchants know

**Comment by eahefnawy** [comment]
okai I will

**Pull request merged**